.gitignore

```diff
@@ -80,7 +80,6 @@ typings/
 
 # Nuxt.js build / generate output
 .nuxt
-dist
```

Comment:
Just curious - do you mean to have the `dist/` directory in the repo?

Reply:
For typescript actions, it turns out that the compiled javascript needs to be committed to the repo ☹️ .  I'm using [ncc](https://github.com/vercel/ncc) to compile everything to an index.js file in the dist folder. (following the pattern set by the [ts action template](https://github.com/actions/typescript-action) from github)